website/src/components/CustomerLogos.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FunctionComponent } from 'react'
+
+interface Logos {
+    name: string
+    src: string
+}
+
+const logos: Logos[] = [
+    {
+        name: 'Amazon',
+        src: '/external-logos/amazon-logo.svg',
+    },
+    {
+        name: 'GE',
+        src: '/external-logos/general-electric-logo.svg',
+    },
+    {
+        name: 'Qualtrics',
+        src: '/external-logos/qualtrics-logo.svg',
+    },
+    {
+        name: 'Dropbox',
+        src: '/external-logos/dropbox-logo.svg',
+    },
+    {
+        name: 'Nutanix',
+        src: '/external-logos/nutanix-logo.svg',
+    },
+    {
+        name: 'Paypal',
+        src: '/external-logos/paypal-logo.svg',
+    },
+    {
+        name: 'Uber',
+        src: '/external-logos/uber-logo.svg',
+    },
+    {
+        name: 'Plaid',
+        src: '/external-logos/plaid-logo.svg',
+    },
+    {
+        name: 'Reddit',
+        src: '/external-logos/reddit-logo.svg',
+    },
+    {
+        name: 'Cloudflare',
+        src: '/external-logos/cloudflare-logo.svg',
+    },
+    {
+        name: 'Mercado Libre',
+        src: '/external-logos/mercado-libre-logo.svg',
+    },
+    {
+        name: 'Lyft',
+        src: '/external-logos/lyft-logo.svg',
+    },
+    {
+        name: 'Adidas Running',
+        src: '/external-logos/adidas-runtastic-sq-logo.svg',
+    },
+    {
+        name: 'Shipt',
+        src: '/external-logos/shipt-logo.svg',
+    },
+]
+
+export const CustomerLogos: FunctionComponent = () => (
+    <div className="d-flex flex-wrap align-items-center justify-content-center max-w-1000 mx-auto user-select-none">
+        {logos.map((logo: Logos) => (
+            <img
```

Comment:
Question, can we just add a global class for `filter: brightness(0);` (`brightness-0` in Tailwind) to apply to these images to treat them b&w, instead of doubling our logo assets? Or is there a reason we wouldn't want to use styling for b&w here?

Reply:
We definitely can and I think that's probably a better approach for many reasons. I definitely thought about this when I first started updating the logos, but now that I've taken the time to manually fix some of the SVG assets, I'm going to go back and update these with the filter for more flexibility. ✨  Thanks Becca!

Reply:
Awesome! 🕺

Reply:
These are all set 🎉